nixos/modules/programs/wayland/sway.nix

```diff
@@ -144,7 +144,19 @@ in
       services.displayManager.sessionPackages = lib.optional (cfg.package != null) cfg.package;
 
       # https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1050913
-      xdg.portal.config.sway.default = lib.mkDefault [ "wlr" "gtk" ];
+      # https://github.com/emersion/xdg-desktop-portal-wlr/blob/master/contrib/wlroots-portals.conf
+      # https://github.com/emersion/xdg-desktop-portal-wlr/pull/315
+      xdg.portal.config.sway = lib.mkDefault {
```

Comment:
Do we overwrite all values when adding any other value?

Reply:
The way I understand `lib.mkDefault`, any change to `xdg.portal.config.sway` would change the whole set. I might be wrong though.

Reply:
Yes. Let's not mark it with `lib.mkDefault` if anyone wants to override its value, they can use `mkForce`.

Reply:
Done in https://github.com/NixOS/nixpkgs/commit/7e7a06994fe41ce961755bb98e168e5c4dd630d4.